portal/apps/org.wso2.carbon.iam.admin-portal.app/src/main/lang/en_US.properties

```diff
@@ -38,3 +39,13 @@ recovery.reset.select.with.offline.passcode=Via Offline Pass Code
 recovery.reset.select.with.email.OTP=Via Email OTP
 recovery.reset.select.with.reset.link=Via Reset Link
 recovery.reset.password.new=New password
+update.header=Update User
+user.profile.update.error=Error occured when updating user profile
+save=Save
+update=Update
+profile.success=Successfully updated the profile!
+profile.failure=An Error occured when updating the profile!
+profile.updated=Your profile is updated.
+invalid.profile.name=Invalid profile name.
+profile.image.loaded=Profile image is loaded
+account.locked=Account is locked till user verify.
```

Comment:
Add a new line.

Reply:
Fix with https://github.com/wso2/product-is/pull/1372